gradoop-flink/src/test/java/org/gradoop/flink/model/impl/operators/matching/single/cypher/planning/queryplan/leaf/FilterAndProjectEdgesNodeTest.java

```diff
@@ -98,8 +99,8 @@ public void testExecute() throws Exception {
     List<Embedding> filteredEdges = node.execute().collect();
 
     assertEquals(1, filteredEdges.size());
-    assertEquals(true, filteredEdges.get(0).getId(0).equals(sourceId));
-    assertEquals(true, filteredEdges.get(0).getId(1).equals(edge1Id));
-    assertEquals(true, filteredEdges.get(0).getId(2).equals(targetId));
+    assertTrue(filteredEdges.get(0).getId(0).equals(sourceId));
```

Comment:
```suggestion
    assertEquals(sourceId, filteredEdges.get(0).getId(0));
```
This would be the cleanest version I think. Please change the other occurrences too.

Reply:
Resolved.